src/exports.jl

```diff
@@ -992,6 +992,7 @@ export minimal_nonfaces
 export minimal_normal_subgroups, has_minimal_normal_subgroups, set_minimal_normal_subgroups
 export minimal_primes
 export minimal_subalgebra_generators
+export minimal_subalgebra_generators_with_relations
```

Comment:
I am (still) opposed to exporting this function under that name.

Reply:
I removed the export.